tests/unit/test_protocol.py

```diff
@@ -240,6 +240,21 @@ async def coro():
 
         protocol._transport.write.assert_called_with(b"config get cluster\r\n")
 
+    async def test_version_command(self, event_loop, protocol):
+        async def coro():
+            return await protocol.version_command()
+
+        task = event_loop.create_task(coro())
+        await asyncio.sleep(0)
+
+        protocol.data_received(b"VERSION 1.6.26\r\n")
+
+        result = await task
+
+        assert result.startswith(b"VERSION")
```

Comment:
I believe that we can assert the full text here. isn't it?

Reply:
I think that the client may have different versions of memcached - https://github.com/memcached/memcached/blob/9dc8b148345b27dd86580641d17f4268885b5894/doc/protocol.txt#L1829

Reply:
I think there may be different versions of memcached. Correct me if I don't see something obvious

Reply:
Oks let me know, once this is changed the PR will be ready to be merged.

Reply:
Well, changes is completed. You can check)